corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -295,31 +295,37 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", ['jquery', 'knockout', 'jquery-ui/u
                 stop: function (e, ui) {
                     ui.item.after($('.selected-for-sort'));
 
-                    // TODO - get working
-                    $('.selected-for-sort').sort(function(a, b) {
-                        var aDataOrder = parseInt(a.attributes['data-order'].value),
-                            bDataOrder = parseInt(b.attributes['data-order'].value);
-                        if (aDataOrder < bDataOrder) {
-                            return -1;
-                        } else if (bDataOrder < aDataOrder) {
-                            return 1;
-                        } else {
-                            return 0;
-                        }
+                    var previousRow = ui.item.prev()[0],
+                        previousIndex = null;
+                    if(previousRow) {
+                        previousIndex = parseInt(previousRow.attributes['data-order'].value);
+                    }
+
+                    var movedIndices = [];
+                    $('.selected-for-sort').map(function (index, element) {
```

Comment:
why are you using `.map` here? I think `.each` might be better for what you are doing.